packages/core/src/request-builder/header-builder/csrf-headers.ts

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
```

Comment:
General: Why did you create a new file `csrf-header` and `auth-headers` to contain the refactored/adjusted code. Why not put the new exported method `getCsrfHeaders` in the old file and deprecate the `addCsrfTokenAndCookies`.

Reply:
Good question, I started by really going function by function and removing everything, and I didn't want everything else to break. I slightly prefer the new names and thought it might be easier to delete this later on also in combination with the according tests, but I see your point - it might be confusing when you want to work on one of those files. What do you think we should do?

Reply:
The argument of removing the files I agree. So for me the approach is fine just harder to see differences.